sos/plugins/openstack_tripleo.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2019 Red Hat, Inc., Cedric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenStackTripleO(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Log installation informations from OpenStack Installer
+    """
+
+    plugin_name = 'openstack_tripleo'
+
+    def setup(self):
+        # Notes: recursion is max 2 for container-puppet and tripleo-config
+        # Those directories are present on all OpenStack nodes and
+        # may contain sensitive data (password).
```

Comment:
This implies `container-puppet` logs can also contain sensitive data, but you dont obfuscate those in `postproc`. Could you please either scrub that passwords, or re-phrase this comment? (or did I mis-interpret the text?)

Reply:
need to rephrase it.